packages/suite/src/views/wallet/coinmarket/common/index.tsx

```diff
@@ -0,0 +1,4 @@
+import { CryptoAmount } from './CryptoAmount';
```

Comment:
no new indexes please, just import it directly

Reply:
That means I won't be able to use the same way as imports e.g. from `@suite-components`:
```
import { Translation, AccountLabeling } from '@suite-components';
```

And that's exactly where I got inspired to avoid this:
```
import { CryptoAmount } from '@wallet-views/coinmarket/common/CryptoAmount';
import { FiatAmount } from '@wallet-views/coinmarket/common/FiatAmount';
```

Is☝️ really the preferred way?

Reply:
Fixed.